### ISSUE TITLE
linux 32 bit support

### ISSUE THREAD

**Issue by frakman1** [cortesi/devd#93]
Hi there,

Thank you for making this available and compiled for so many platforms.

Can I ask that you provide a 32-bit linux support as well?  I am using the below version of (k)Ubuntu
```
$uname -a
Linux kubuntu14 4.4.0-59-generic #80~14.04.1-Ubuntu SMP Fri Jan 6 18:02:23 UTC 2017 i686 i686 i686 GNU/Linux
```

Also, I couldn't find your build scripts or compile instructions as  pervious link to `build scripts` was broken.

**Comment by cortesi** [comment]
Hi there - I've just released v0.9, which includes a 32-bit Linux build. Thanks!

**Comment by frakman1** [comment]
Nice! Thank you.
